src/storageManager.js

```diff
@@ -167,13 +176,26 @@ export function newStorageManager({gvlid, moduleName, bidderCode, moduleType} =
     }
   }
 
+  /**
+   * @param {string} consentData
+   */
+  const hasPurposeOneConsent = function(consentData) {
```

Comment:
Hi @renebaudisch 

There is already a similar function in `src/utils/gpdr.js` under the name `hasPurpose1Consent`.  Would you be able to use that function instead of redefining it here?